VC/instrumentation/read_labjack.py

```diff
@@ -0,0 +1,268 @@
+  rate possible. All channels and their settings for each sensor are 
+  configured here. Data produced by the LJ is consumed and written to
+  a file for later processing.
+
+
+Sensor Overview:
+
+  Both the CTL-190 and HKM-375 are absolute pressure transducers that have
+  a supply voltage of 10VDC, output impedance of 1000 ohms and a differential
+  output with full scale range of 100mV.
+
+  All thermocouples used are K-type.
+
+  Load cells used are typical bridge sensors with low output impedance and
+  a very small full scale range. Supplied with 5VDC.
+
+
+streamConfig():
+
+  Streaming data from the LJ is the fastest way to acquire data.
+  See section 5.2.12 for StreamConfig documentation for details.
+
+  samples_per_packet:
+    The number of samples in each packet sent over USB by the LJ. If the
+    number of channels being scanned is low, then this number should be
+    set lower than the default of 25, otherwise the packet frequency will
+    be lower than desirable. If the number of channels is high then the number
+    should be left at 25 or increased.
+
+  scan_frequency:
+    The frequency in Hz to scan the channel list (ChannelNumbers). 
+    The sample rate (Hz) = scan_frequency * num_channels.
+
+  num_channels:
+    Number of channels to stream. Equal to length of channel_numbers.
+
+  resolution_index:
+    See section 3.1. Higher values increase ENOB but at the expense
+    of longer sample times.
+
+  settling_factor:
+    For high impedance sensors, a longer settling time is required to acquire
+    enough charge in the ADC to get an accurate reading. LJ says that the
+    auto setting is good for any sensor with at most 1000 ohm output 
+    impedance. 
+    See https://support.labjack.com/docs/analog-input-settling-time-app-note
+ 
+  channel_settings [0, 1]:
+    Channels to read from and their settings. For differential pairs, only
+    specifiy the positive channel (even numbered). OR a channel mode and 
+    gain value together to conveniently set the options for that channel.
+
+    Each channel has options that can be set with the ChannelOptions byte.
+    These options should be configured for each type of sensor to account
+    for their output impedance, fullscale range etc. Each bit of the byte
+    configures a different setting:
+
+      Set bit 4-5 = GainIndex:
+        GainIndex   Gain   Max V    Min V
+        b00         1      10.1     -10.6
+        b01         10     1.01     -1.06
+        b10         100    0.101    -0.106
+        b11         1000   0.0101   -0.0106
+    
+      Set bit 7 for differential reading.
+
+    Ex:
+      Channel 0 single ended with gain 10:  (0, SING | X10)
+      Channel 0,1 differential with gain 1: (0, DIFF | X1)
+
+'''
+
+######### BEGIN USER ADJUSTABLE #########
+
+# Stream settings
+scan_frequency   = 25
+resolution_index = 2
+settling_factor  = 2
+samples_per_packet = 50
+channel_settings = [(86, DIFF | X1000), # L_RUN_TANK (SEEMS GOOD. CHECK CAL)
```

Comment:
Maybe we could create a step by step guide on how to add a new sensor. Probably scope creep for this PR, Ill make a follow up ticket

Reply:
There is some stuff in the big comment about how to do it but I can flesh that out a bit more so its more obvious. It'll only need to be like 2 sentences.